src/rich-text/commands/index.ts

```diff
@@ -313,7 +313,7 @@ export function insertRichTextLinkCommand(
                 state.selection.content().content.firstChild?.textContent ??
                 null;
             const linkMatch = /^http(s)?:\/\/\S+$/.exec(selectedText);
-            linkUrl = linkMatch?.length > 0 ? linkMatch[0] : "";
+            linkUrl = linkMatch?.length > 0 ? linkMatch[0] : "https://";
```

Comment:
I'd find this a little less convenient because I'd paste a link here most often, so having the input prepopulated with `https://` adds a step of deleting the text. Instead, my suggestion is to use the [`placeholder` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/placeholder) with a value of `https://…` or something like "Add a link" similar to Google Docs:

![image](https://github.com/user-attachments/assets/2957a1cc-3eb2-4be6-8963-d562de86bb39)

I don't think the placeholder is entirely necessary as long as the Link URL field is focused upon invoking the link editor. 

All this said, I'd like to get others' opinions on this since it's entirely possible this is just a _me_ thing.

Reply:
I agree with @dancormier. I don't think it is a good practice to prefill text input. Placeholders are a more conventional way of giving hints to users. I would recommend to check with your designer about this whole flow and seek their input. 🙂